ntbs-service/DataAccess/ReferenceDataRepository.cs

```diff
@@ -183,17 +183,22 @@ public async Task<List<string>> GetPhecCodesMatchingRolesAsync(IEnumerable<strin
 
         public IQueryable<TBService> GetTbServicesQueryable()
         {
-            return _context.TbService;
+            return _context.TbService.OrderBy(s => s.Name);
```

Comment:
If we've got this TBServiceQueryable can we use this in GetTbServiceByCodeAsync and GetAllTbServicesAsync? Could be worth renaming to GetOrderedTbServicesQueryable too but that might be overkill.